### DISCUSSION TITLE
Rename `packages` folder to `src`

### DISCUSSION THREAD

**Discussion by talks2much** [cawa-93/vite-electron-builder#300] [Q&A] [answered]
Why is that called **packages**?? This isn't a monorepo for NPM packages so why don't we call it **src**?

**Comment by cawa-93** [comment, marked as answer]
>Why is that called packages?

You can call it `modules` if you want. It's just a word. I chose `packages` because my editor provided a nice icon for directory  called `packages`. 

> why don't we call it src?

Because each "package" (or, "module") contains `src` itself. It's just a little clearer to me.

You can take a look [old file structure](https://github.com/cawa-93/vite-electron-builder/tree/3fbfb2ce0cd00fdebc20ac339563382dc287f5bb). Here are related [discussion](https://github.com/cawa-93/vite-electron-builder/discussions/150) and [PR](https://github.com/cawa-93/vite-electron-builder/pull/158)
